table_engine/src/table.rs

```diff
@@ -318,13 +319,17 @@ pub struct ReadOptions {
     /// Suggested read parallelism, the actual returned stream should equal to
     /// `read_parallelism`.
     pub read_parallelism: usize,
+    /// Request deadline
+    pub deadline: Instant,
 }
 
 impl Default for ReadOptions {
     fn default() -> Self {
         Self {
             batch_size: 10000,
             read_parallelism: DEFAULT_READ_PARALLELISM,
+            // TODO: read from config
+            deadline: Instant::now() + Duration::from_secs(60),
```

Comment:
Maybe `None` is better for the default deadline denoting no timeout.

Reply:
fixed